spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarTemplateTests.java

```diff
@@ -60,18 +61,67 @@
  */
 class PulsarTemplateTests extends AbstractContainerBaseTests {
 
-	private static final String SAMPLE_MESSAGE_KEY = "sample-key";
+	@ParameterizedTest(name = "{0}")
+	@MethodSource("interceptorInvocationTestProvider")
+	void interceptorInvocationTest(String topic, List<ProducerInterceptor> interceptors) throws Exception {
```

Comment:
Sorry for the formatting / moving around in the same PR / commit. I will call out what has changed and what is the same to make this easier to review....

**[UNCHANGED-JUST-MOVED-UP]**